core/app/modules/kit-library/assets/js/pages/index/index.js

```diff
@@ -228,26 +229,16 @@ export default function Index( props ) {
 							isSuccess && 0 === data.length && queryParams.ready && props.renderNoResultsComponent( {
 								defaultComponent: <ErrorScreen
 									title={ __( 'No results matched your search.', 'elementor' ) }
-									description={ __( 'Try different keywords or continue browsing.', 'elementor' ) }
+									description={ __( 'Try different keywords or ', 'elementor' ) }
```

Comment:
Why?

Reply:
The 'continue browsing' is a button next to the description.